### PR TITLE
Add gRPC API for exitcode

### PR DESCRIPTION
**Pull request by Crazykev**
Add gRPC API for exitcode
part of #278 

Signed-off-by: Crazykev <crazykev@zju.edu.cn>

### CODE REVIEW

File: integration/client.go [hunk 1]
```diff
@@ -41,6 +41,20 @@ func (c *HyperClient) GetPodInfo(podID string) (*types.PodInfo, error) {
 	return pod.PodInfo, nil
 }
 
+// ExitCode gets exitcode by container and tag
+func (c *HyperClient) ExitCode(container, tag string) (int32, error) {
+	request := types.ExitCodeRequest{
+		Container: container,
+		Tag:       tag,
```
Comment (laijs) [inline integration/client.go]: tag is going to be removed

Comment (Crazykev) [inline integration/client.go]: @laijs So how can we decide which tty without a client tag?

Comment (laijs) [inline integration/client.go]: ExitCode is for process(container's init process or exec process). the tag will be replaced by process Id.

Comment (Crazykev) [inline integration/client.go]: got it. I'll update this patch until related patch is merged.

Comment (feiskyer) [inline integration/client.go]: @Crazykev we can make a workaround for now: Rename `ExitCode` to `Wait`, and rename `tag` to id.

Comment (Crazykev) [inline integration/client.go]: yep, updated and rebased @feiskyer

File: types/types.proto [hunk 2]
```diff
@@ -366,10 +366,20 @@ message ContainerLogsRequest {
   bool    stderr      = 7;
 }
 
+
```
Comment (laijs) [inline types/types.proto]: why there is an additional empty line here?

File: types/types.proto [hunk 3]
```diff
@@ -366,10 +366,20 @@ message ContainerLogsRequest {
...
 message ContainerLogsResponse {
   bytes log = 1;
 }
 
+message WaitRequest{
+    string container = 1;
+    string processId = 2;
+}
```
Comment (laijs) [inline types/types.proto]: it seems better if there is a `bool noHang = 3`.
it can be added later if it is needed.

**Comment by Crazykev** [comment]
@laijs updated and rebased

**Pull request closed without merging**